R/download.R

```diff
@@ -23,45 +25,53 @@ download_bugzilla_issues_from_rest_api <- function(bugzilla_site, start_timestam
     save_folder_path <-paste0(save_folder_path, "/")
   }
 
+  # Holds the bug ids
+  bug_ids <- c()
+  # Defines what bug to start from in bugs retrieved.
+  offset <- 0
+  # Defines name of the file. Each page contains 500 bugs.
+  page <- 0
+
   # Get the issues starting from the specified date
-  issues <- httr::GET(paste0(bugzilla_site, "/bug", "?creation_time=", start_timestamp, "&limit=20", "&offset=0"),
-                      httr::write_disk(file.path(paste0(save_folder_path, 0, ".json")), overwrite = TRUE))
+  issues <- httr::GET(paste0(bugzilla_site, "/bug", "?creation_time=", start_timestamp, "&limit=", limit, "&offset=", offset))
```

Comment:
I use write_json instead of httr::write_disk so that I can check if the json data retrieved from a GET request is empty, so I know whether I should write the json file. Before, I was writing an extra empty json file at the end (since I wasn't able to immediately tell if the JSON was empty), but separating the GET request and writing the file fixes this issue.

Reply:
Sounds good!